Jenkinsfile

```diff
@@ -39,9 +39,21 @@ onlyOnMaster {
     }
 
     milestone()
-    stage("prod deploy") {
-        input(message: "Deploy to prod?")
-        milestone()
-        deployApp(image: img, app: "h-periodic", env: "prod", region: "us-west-1")
+    stage("approval") {
+        input(message: "Proceed to production deploy?")
+    }
```

Comment:
Dedicated approval stage has been introduced to make it easier to understand the Jenkins stage view at a glance, and any user input that might be required. The example below has been taken from a similar workflow (via)

<img width="798" alt="Screenshot 2021-06-28 at 11 00 52" src="https://user-images.githubusercontent.com/18003117/124463228-7b5f4a80-dd8a-11eb-9076-02d5d509df3e.png">